Python/Product/Profiling/PythonProfilingPackage.cs

```diff
@@ -303,9 +316,59 @@ protected override int CreateToolWindow(ref Guid toolWindowType, int id)
             config.ScriptArguments = runTarget.Arguments;
             config.WorkingDirectory = runTarget.WorkingDirectory;
 
-            RunProfiler(session, config, openReport);
+#if EXTERNAL_PROFILER_DRIVER
+            if (useVTune) {
+                RunVTune(session, config, openReport);
+            } else {
+#endif
+                RunProfiler(session, config, openReport);
+#if EXTERNAL_PROFILER_DRIVER
+            }
+#endif
         }
 
+#if EXTERNAL_PROFILER_DRIVER
+        private static void RunVTune(SessionNode session, LaunchConfiguration config, bool openReport) {
+
+            var interpreter = config.GetInterpreterPath();
+            if (!File.Exists(interpreter)) {
+                MessageBox.Show(Strings.CannotFindPythonInterpreter, Strings.ProductTitle);
+                return;
+            }
+
+            string outPathDir = Path.GetTempPath();
```

Comment:
This `outPathDir` needs it own unique subfolder. You use it below `Path.Combine(outPathDir, "Sample.dwjson")`, so that has to be unique per run.

I see you've made changes elsewhere to generate datetime based subfolder, I don't know if those were necessary or not. Also note that using Path.GetRandomFileName may be simpler than all the date formatting, unless you find that useful to track down the files in the temp folder.
Example from TestData.CalculateTempRoot
```
                path = Path.GetTempPath();
                var subpath = Path.Combine(path, Path.GetRandomFileName());
                while (Directory.Exists(subpath) || File.Exists(subpath)) {
                    subpath = Path.Combine(path, Path.GetRandomFileName());
                }
                path = subpath;
```